### DISCUSSION TITLE
Warning as repository isn't found

### DISCUSSION THREAD

**Discussion by adriandoerfler** [jiangcuo/Proxmox-Port#78] [General]
After running 
`sudo curl -L https://mirrors.apqa.cn/proxmox/debian/pveport.gpg -o /etc/apt/trusted.gpg.d/pveport.gpg`
and
`echo "deb [deb=arm64 signed-by=/usr/share/keyrings/pveport.gpg] https://mirrors.apqa.cn/proxmox/debian/pve bookworm port" | sudo tee  /etc/apt/sources.list.d/pveport.list
`
`sudo apt update`
Still creates an error:
`sudo apt update
Hit:1 http://deb.debian.org/debian bookworm InRelease
Hit:2 http://deb.debian.org/debian-security bookworm-security InRelease     
Hit:3 http://archive.raspberrypi.com/debian bookworm InRelease                                   
Hit:4 http://deb.debian.org/debian bookworm-updates InRelease                                    
Hit:5 https://de.mirrors.apqa.cn/proxmox/debian/pve bookworm InRelease     
Reading package lists... Done                        
Building dependency tree... Done
Reading state information... Done
All packages are up to date.
W: Skipping acquire of configured file 'port/binary-armhf/Packages' as repository 'https://de.mirrors.apqa.cn/proxmox/debian/pve bookworm InRelease' does not seem to provide it (sources.list entry misspelt?)`

Is there a fix to it?

**Comment by jiangcuo** [comment]
Use "de.mirrors.apqa.cn"  instead of "mirrors.apqa.cn".Probably an issue with Cloudflare redirects

**Reply by adriandoerfler** [reply]
Thanks for the reply.
Unfortunately I was not able to resolve the issue:
`W: Skipping acquire of configured file 'port/binary-armhf/Packages' as repository 'https://de.mirrors.apqa.cn/proxmox/debian/pve bookworm InRelease' does not seem to provide it (sources.list entry misspelt?)`

**Comment by tacticz** [comment]
Probably a bit late but I think that using `[arch=arm64]` should solve this problem:

```
echo 'deb [arch=arm64] https://mirrors.apqa.cn/proxmox/debian/pve bookworm port' > /etc/apt/sources.list.d/pveport.list
```

**Reply by luisfalvarez-sh** [reply]
Thank you bro
